build-system/common/git.js

```diff
@@ -180,7 +222,7 @@ function gitCommitterEmail() {
  * @return {!Array<{sha: string, isCherryPick: boolean}>}
  */
 function gitCherryMaster() {
-  return getStdout('git cherry master')
+  return getStdout(`git cherry ${gitFindMaster()}`)
```

Comment:
This won't work because `git cherry` takes a commit or a local branch name, but not an alias for a remote branch. I think you'd need to determine the SHA corresponding to `upsteram/master` for this to work.

![image](https://user-images.githubusercontent.com/26553114/80536030-bd635400-896f-11ea-86f4-3dcd69595a8e.png)

Reply:
Something seems off here (and definitely doesn't match my experience). Let me look into this to see why `git cherry` doesn't associate `upstream/master` with a sha for you. Could you let me know your git version?

Reply:
Standard version that shipped with my 2017 Macbook.

![image](https://user-images.githubusercontent.com/26553114/80537298-cb19d900-8971-11ea-89d5-f51d7c96fce9.png)